Assets/Scripts/MainUIController.cs

```diff
@@ -32,9 +32,22 @@ public class MainUIController : MonoBehaviour
     // city selection controls
     private CityDropdown cityDropdown;
 
+    // handle sphere interaction
+    private GameObject sphere;
+    private float moveSpeed = .1f;
+    private float scaleSpeed = 5f;
+    private float maxScale = 100f;
+    private float minScale = .025f;
+    private float rotateSpeed = 10f;
+    private float autoRotateSpeed = 1f;
+    private bool rotating = false;
+    private MarkersController markersController;
+
     // Start is called before the first frame update
     void Start()
     {
+        sphere = GameObject.Find("CelestialSphere");
```

Comment:
doesn't the DataController have a reference to this object? Get it from there, instead of Find, and use the singleton GetInstance I suggested instead of that find.

Reply:
I always add these in because I like them when testing, but I agree, let's not rely on this pattern since we're multi-platform.  Removed.

Reply:
It's the GameObject that the DataController is attached to so all we need is `dataController.gameObject`.  Fixed.